sale_cancel_reason/__openerp__.py

```diff
@@ -19,15 +20,17 @@
 #
 #
 
-{'name': 'Sale Cancel Reason',
- 'version': '8.0.1.1',
- 'author': "Camptocamp,Odoo Community Association (OCA)",
- 'category': 'Sale',
- 'license': 'AGPL-3',
- 'complexity': 'normal',
- 'images': [],
- 'website': "http://www.camptocamp.com",
- 'description': """
+{
+    'name': 'Sale Cancel Reason',
+    'version': '9.0.1.0.0',
+    'author': 'Camptocamp, Odoo Community Association (OCA),'
+            ', Serpent Consulting Services Pvt. Ltd.',
+    'category': 'Sale',
+    'license': 'AGPL-3',
+    'complexity': 'normal',
+    'images': [],
+    'website': "http://www.camptocamp.com",
+    'description': """
 Sale Cancel Reason
```

Comment:
Put description in a README.rst according conventions (https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst)